http-add-on/values.yaml

```diff
@@ -193,6 +193,15 @@ interceptor:
     # -- Port that the interceptor proxy TLS server should be started on
     port: 8443
 
+  # configuration of pdb for the interceptor
+  pdb:
+    # -- Whether to install the `PodDisruptionBudget` for the interceptor
+    enabled: false
```

Comment:
Any reasons why we would not enable this by default?

Reply:
I wasn't sure whether or not we were okay with a change that would modify the default installation. I can default to enabled!